evap/evaluation/templates/base.html

```diff
@@ -110,7 +110,8 @@
                 language: "{{ LANGUAGE_CODE }}",
                 placeholder: "{% trans 'Please select...' %}",
                 minimumResultsForSearch: 15,
-                minimumInputLength: $(this).children('option').length > 50 ? 3 : 0
+                minimumInputLength: $(this).children('option').length > 50 ? 3 : 0,
+                allowClear: (this.name == "participants") ? true : false
```

Comment:
I'd suggest adding a second argument to this function (`applySelect2`) that allows to provide additional arguments or override some of the default ones. Then, the place where the "exceptional" behaviour is defined is closer to where we actually need it.

(afaik javascript has some object packing and unpacking semantics similar to **kwargs in python)

Reply:
i would replace this line with `allowClear: this.multiple` instead. it's actually helpful to add this for all multi select fields and we don't have to handle it individually.